addons/mail/static/src/models/follower_list_menu_view.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'FollowerListMenuView',
+    identifyingFields: [['chatterOwner']],
```

Comment:
Why the double `[]` ?

Reply:
It's for easing the shenanigan with mocking owner of `FollowerListMenuView` record in test, so that the bad tests don't need to be adapted immediately with this PR.

The advantage of this trick in tests is to adapt bad tests later...
...But this quickly prevent some other changes, as sooner or later the tests must be adapted...

A follow-up PR is blocked because of this:
https://github.com/odoo/odoo/pull/88684